README.md

```diff
@@ -1921,22 +2096,31 @@ Plot.map({y: "cumsum"}, {y: d3.randomNormal()})
 
 Groups on the first channel of *z*, *fill*, or *stroke*, if any, and then for each channel declared in the specified *outputs* object, applies the corresponding map method. Each channel in *outputs* must have a corresponding input channel in *options*.
 
-#### Plot.mapX(*map*, *options*)
+
+<!-- jsdoc mapX -->
+
+#### Plot.mapX(*mapping*, *options*)
```

Comment:
here also we have renamed an argument of the API, can we do the opposite?

Reply:
In this case I couldn't because using the name `map` would shadow the `map` function that is called internally. I wanted to avoid `m` as well as aliasing `map` in the map file. Maybe there is a better name than `mapping`?

Reply:
Ah yes, it makes sense—this map function is exposed, and it would probably be awkward to rename it internally to (say) mapT, the export mapT as map. *mapping* is probably fine.

Reply:
another possibility would be *output* (singular), to underscore that it is just one element of the more generic outputs of the map transform

Reply:
note that it also has a link (in the Plot.normalize section) that we'll have to change